### ISSUE TITLE
Disqus not working on my host 

### ISSUE THREAD

**Issue by anixer0** [KirixenYT/Gogoanime#9]
What file a I need to edit to work the disqus

**Comment by KirixenYT** [comment]
> What file a I need to edit to work the disqus

All u need is a link of ur disqus shortname and i'll fix the problem of it not working

**Comment by anixer0** [comment]
This link? 
.
.
https://animxer-7.disqus.com/

**Comment by anixer0** [comment]
Thanks now I know what file I need to edit

**Comment by anixer0** [comment]
Now it's work.

https://anixer.000webhostapp.com/chibi-maruko-chan-episode-1435

**Comment by KirixenYT** [comment]
> Now it's work.
> 
> https://anixer.000webhostapp.com/chibi-maruko-chan-episode-1435

Congrats man, I'll try to fix the disqus to show according to the website

**Comment by anixer0** [comment]
Bro if you need hosting site try https://pawnhoster.com

**Comment by anixer0** [comment]
![Screenshot_20240624-200121](https://github.com/KirixenYT/Gogoanime/assets/172897096/a8d69d64-3650-4f61-8924-e73890d50a4a)
Here's the pic of free plan hosting on there site..

**Comment by KirixenYT** [comment]
> Bro if you need hosting site try https://pawnhoster.com

THANK YOU MAN :), I needed that since i got like suspended on most accounts

**Comment by KirixenYT** [comment]
> Bro if you need hosting site try https://pawnhoster.com

but isn't it for wordpress only and i need to pay for the domain?

**Comment by anixer0** [comment]
> > Bro if you need hosting site try https://pawnhoster.com
> 
> but isn't it for wordpress only and i need to pay for the domain?

Yup you need to buy... But if you have existing domain name you can add to there hosting by using nameser

**Comment by anixer0** [comment]
If you want to buy domain name.. for cheap price visite hostinger site only 1$

**Comment by KirixenYT** [comment]
> If you want to buy domain name.. for cheap price visite hostinger site only 1$

I see and thanks for sharing it

**Comment by ArjixWasTaken** [comment]
> If you want to buy domain name.. for cheap price visite hostinger site only 1$

hostinger charges a lot after the 1-2 year period of the offer
you get a "free domain" and after that it is like 20 euro per year or smth

sure that ain't pricy, but I recommend actual domain resellers and not getting free domains
